.editorconfig

```diff
@@ -16,16 +18,12 @@ end_of_line = lf
 # Formatting - remove any whitespace characters preceding newline characters
 trim_trailing_whitespace = true
 
-[*.cs]
-
-charset = utf-8
-
-# Core editorconfig formatting - indentation
+[*.xaml]
 
-# use hard tabs for indentation
-indent_style = tab
+indent_style = space
```

Comment:
We have a de-facto indentation style in `.xaml` file But I am not sure if that is good or not. IMO we should have the same style across allthe projects' files.

Reply:
What style is that?